crates/libcontainer/src/rootfs/mount.rs

```diff
@@ -378,7 +374,7 @@ impl Mount {
         if let Some(l) = label {
             if typ != Some("proc") && typ != Some("sysfs") {
                 match mount_option_config.data.is_empty() {
-                    true => d = format!("context=\"{}\"", l),
+                    true => d = format!("context=\"{l}\""),
                     false => d = format!("{},context=\"{}\"", mount_option_config.data, l),
```

Comment:
Hey, why is this part not formatted in the same way? we can format `l` in the same way here right?

Reply:
I'm not sure because this changes created by cargo fmt and clippy